src/hooks/use-expiration-date.ts

```diff
@@ -47,6 +63,7 @@ export function useExpirationDate( snapshotDate: number ) {
 	return {
 		isExpired,
 		countDown: isExpired ? __( 'Expired' ) : countDown,
+		fullDateString: formatStringDate( endDate.getTime(), locale, 'long' ),
```

Comment:
Should we call it `expireDateString` or `fullExpireDateString` or `endDateString` ?

```suggestion
		expireDateString: formatStringDate( endDate.getTime(), locale, 'long' ),
```

Reply:
That's a valid point. I named the string based on the existing `dateString` we are already returning by `useExpirationDate` when used in the current Demo sites UI.

But since we are going to remove that UI soon, I think it makes sense to improve the naming of `fullDateString`. 

`expireDateString` sounds the most clear to me.

Reply:
→ Updated in https://github.com/Automattic/studio/pull/919/commits/c85f3d276bc55e8b6606ad0779532e98ba5599db.